src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,39 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+class Block{
+	public static int blockNumber = 123;
```

Comment:
Why is the blockNumber class variable state? When a class variable is static it will be the same value for all of the class instances, which is incorrect.